django-stubs/contrib/admin/options.pyi

```diff
@@ -279,6 +279,9 @@ class ModelAdmin(BaseModelAdmin[_ModelT]):
     def history_view(
         self, request: HttpRequest, object_id: str, extra_context: dict[str, Any] | None = ...
     ) -> HttpResponse: ...
+    def get_formset_kwargs(
+        self, request: HttpRequest, obj: _ModelT, inline: InlineModelAdmin, prefix: str
```

Comment:
ah wait a sec, the `InlineModelAdmin` needs type parameters. Should it be: `InlineModelAdmin[Any, _ModelT]` ?

Reply:
By `Any` I meant `Model`, as in "any model", because all inlines are iterated over and passed to this function. Changed.

Reply:
We've used `Any` instead of `Model` in places where users expect a concrete model instance with its fields to be present, but inference doesn't know what model exactly.

Here it seems less likely that users need to interact with `InlineModelAdmin`'s model, but I'm not sure, I couldn't find many exaples on the web. I would be fine with either.